README.md

```diff
@@ -116,6 +117,12 @@ iota-sdk = { git = "https://github.com/iotaledger/iota-sdk" branch = "develop" }
 
 ## Client Usage
 
+The following example creates a [`Client`](https://docs.rs/iota-sdk/latest/iota_sdk/client/index.html)
```

Comment:
Link to the struct ? https://docs.rs/iota-sdk/latest/iota_sdk/client/core/struct.Client.html

Reply:
```suggestion
The following example creates a [`Client`](https://docs.rs/iota-sdk/latest/iota_sdk/client/core/struct.Client.html)
```